.circleci/config.yml

```diff
@@ -93,14 +93,14 @@ references:
     restore_cache:
       key: *build_cache_key
 
-  machine_build_cache_key: &machine_build_cache_key machine-build-cache-v1-{{ .Branch }}-{{ .Revision }}
+  machine_build_cache_key: &machine_build_cache_key machine-build-cache-v2-{{ .Branch }}-{{ .Revision }}
```

Comment:
This is a workaround suggested by @dincho to avoid using CircleCI cache, and start fresh when fetching deps for system tests.

Hackney is defined as dependency for system tests only, and is not included in `rebar.lock` (see https://github.com/aeternity/epoch/blob/master/rebar.config#L139 for reference). I failed to find a reasonable way of forcing hackney to upgrade it (even `@./rebar3 as system_test upgrade` only upgrades lib in `_build/system_test`, and `_build/system_test+test` remains untouched).

Locally, it is best to just remove content of `_build/system_test` and `_build/system_test+test` dirs.

Reply:
> This is a workaround suggested by @dincho to avoid using CircleCI cache, and start fresh when fetching deps for system tests.

It actually does not avoid the cache, but invalidates the old (v1) caches. Effectively means that after this is merged all PR's will start using the new (machine) cache.

Reply:
Exactly, this is what I meant, I was not clear enough :)